tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/MetadataConnectionProvider.java

```diff
@@ -38,4 +42,9 @@ public void disconnect(MetadataConnection metadataConnection) {
   public ConnectionValidationResult validate(MetadataConnection metadataConnection) {
     return ConnectionValidationResult.success();
   }
+
+  @Override
+  public void start() throws MuleException {
+    STARTED = true;
```

Comment:
shouldn't this be set to false on the test setup to prevent false positives from previous that also use this class?